### PR TITLE
chore: check for release ticket on release start

### PR DESCRIPTION
**Pull request by mcasimir**
- Check that a Release ticket exists and is in progress before doing a GA
- Remind to move the ticket to done afterwards

### CODE REVIEW

File: packages/compass/release/commands.js [hunk 1]
```diff
@@ -55,19 +58,23 @@ async function startRelease(bumpFn, evergreenProject) {
...
+    `Are you sure you want to bump from ${chalk.bold(packageJsonVersion)} ` +
+      `to ${chalk.bold(newSemver)} and release?`
   );
 
   if (!answer) {
     cli.info('cancelled.');
     return;
   }
 
+  isReleaseTicketInProgress(newSemver);
```
[the remark this answers is not in the document]
Comment (mcasimir) [reply packages/compass/release/commands.js:70]: yup, weird that tests didn't catch this, maybe we don't run them in CI .. have to check

File: packages/compass/release/jira.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+function buildFetchJqlUrl(jiraApiBaseUrl, jql) {
+  const url = new URL(`${jiraApiBaseUrl}/rest/api/2/search`);
+  url.searchParams.append('jql', jql);
+  url.searchParams.append('fields', 'key,status,summary');
+  url.searchParams.append('maxResults', '1');
+  return url.toString();
+}
+
+const releaseVersionJql = (releaseVersion) =>
+  `project = COMPASS AND fixVersion = ${releaseVersion} AND issuetype = Release`;
```
[the remark this answers is not in the document]
Comment (mcasimir) [reply packages/compass/release/jira.js:18]: that's better thanks!

File: packages/compass/release/commands.js [hunk 3]
```diff
@@ -55,19 +58,23 @@ async function startRelease(bumpFn, evergreenProject) {
...
+    `Are you sure you want to bump from ${chalk.bold(packageJsonVersion)} ` +
+      `to ${chalk.bold(newSemver)} and release?`
   );
 
   if (!answer) {
     cli.info('cancelled.');
     return;
   }
 
+  checkReleaseTicketInProgress(newSemver);
```
Comment (addaleax) [inline packages/compass/release/commands.js:70]:
```suggestion
  awaot checkReleaseTicketInProgress(newSemver);
```

**Review by addaleax — APPROVED** [review]

File: packages/compass/release/commands.js (hunk 3 above)
Comment (mcasimir) [reply packages/compass/release/commands.js]: OMG 🤦

**Pull request merged**